### DISCUSSION TITLE
rebase and merge: select commiter email

### DISCUSSION THREAD

**Discussion by eponier** [community/community#49129] [Pull Requests]
### Select Topic Area

Bug

### Body

Since september 2022 (https://github.blog/changelog/2022-09-15-git-commit-author-shown-when-squash-merging-a-pull-request/), the "squash & merge" merging method allows to select the author email, among the ones the user has configured, if the user merging the PR is the one who opened it. The "merge commit" merging method also allows to select the email that will be used for the merge commit.

The third merging method, "rebase and merge", automatically fills the "commit" field of the commits in the PR with the primary address of the user who merges the PR using this method. There is no way to select the address the user wants to use. It would be nice to have a checkbox to choose the right address, and, like in the other cases, a reasonable default (the email address used in the commits or the email address used for notifications in case of an organization). This would allow to consistently use the same email for all operations inside an organization. Currently, the (potentially personal) primary address of the user is used in "rebase and merge", which creates some inconsistency.

**Comment by ivanperez-keera** [comment]
This also happens for the COMMITTER email when one rebases. I'd be great if we could choose.
